src/basic/log.c

```diff
@@ -1357,6 +1358,9 @@ int log_syntax_internal(
                 const char *func,
                 const char *format, ...) {
 
+        if (level == LOG_ERR || level == LOG_WARNING)
```

Comment:
We'd write this as `IN_SET(level, LOG_ERR, LOG_WARNING)`. But selecting just those two levels seems very strange. Why not `level <= LOG_WARNING` ?